oci/tests/versions_test.bzl

```diff
@@ -7,7 +7,7 @@ load("//oci/private:versions.bzl", "CRANE_VERSIONS")
 
 def _smoke_test_impl(ctx):
     env = unittest.begin(ctx)
-    asserts.equals(env, "v0.11.0", CRANE_VERSIONS.keys()[0])
+    asserts.equals(env, "v0.0.0-nightly", CRANE_VERSIONS.keys()[0])
```

Comment:
opt: we should delete this test, it's a placeholder for real testing of custom rules when we have them